ergo/scale.py

```diff
@@ -33,10 +33,10 @@ def denormalize_point(self, point):
         return (point * self.width) + self.low
 
     def denormalize_points(self, points):
-        return [self.denormalize_point(point) for point in points]
+        return np.array([self.denormalize_point(point) for point in points])
```

Comment:
Should be vectorized. Same applies to `normalize_points`, `denormalize_densities`, and `normalize_densities`, both here and in `LogScale`.